tests/main/canonical-livepatch/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Ensure canonical-livepatch snap works
+
+# livepatch works only on 16.04 amd64 systems
+systems: [ubuntu-16.04-64]
+
+execute: |
+    echo "Ensure canonical-livepatch can be installed"
+    snap install canonical-livepatch
+    wait_for_service snap.canonical-livepatch.canonical-livepatchd.service
```

Comment:
don't you need to source `systemd.sh` for this to work?

Reply:
Silly me, indeed